src/pages/home/report/ReportActionItemParentAction.js

```diff
@@ -41,6 +42,11 @@ const defaultProps = {
 
 const ReportActionItemParentAction = (props) => {
     const parentReportAction = props.parentReportActions[`${props.report.parentReportActionID}`];
+
+    // In case of transaction threads, we do not want to render the parent report action.
+    if (ReportActionsUtils.isTransactionThread(parentReportAction)) {
+        return null;
+    }
```

Comment:
This is to ensure we do not show any report Action at the top of the transactions thread

Reply:
I'm working on a change that might affect this line, can you clarify what the goal here was? @mountiny